src/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,35 @@
+import { RiAddLine, RiTodoLine } from "@remixicon/react";
+import SidebarLinks from "./components/sidebar-links/sidebar-links";
+import styles from "./sidebar.module.css";
+import Modal from "../ui/modal/modal";
+import CreateTodo from "../ui/create-todo/create-todo";
+import { useModalStore } from "../../store/modal/modal-store";
+import Button from "../ui/button/button";
+export default function Sidebar() {
+  const links = [
+    {
+      label: "Your Tasks",
+      link: "/",
+      icon: <RiTodoLine size={20} color="white" />,
+    },
+    {
+      label: "Completed Tasks",
+      link: "/completed-tasks",
+      icon: <RiTodoLine size={20} color="white" />,
+    },
+  ];
```

Comment:
Since this constant doesn't depend on the component state, it should be declared outside the component.